### DISCUSSION TITLE
Version 5.4.2 API documentation

### DISCUSSION THREAD

**Discussion by rockfordlhotka** [MarimerLLC/csla#2455] [Announcements]
Hello everyone, I am excited to announce that (thanks to work by @Brannos1970) there are now online API docs available for CSLA version 5.4.2.

You can see the [CSLA version 5.4.2 API documentation](https://cslanet.com/5.4.2/html/index.html) online.

**Comment by rockfordlhotka** [comment]
@Brannos1970 - you will be happy to hear that people are using the docs - over 80 visits in the past couple weeks!

**Reply by Brannos1970** [reply]
That is great to hear. When version 6 is ready I can generate the documentation.
